tgui/packages/tgui/interfaces/CommandTablet.js

```diff
@@ -9,13 +9,32 @@ export const CommandTablet = (_props, context) => {
 
   const AlertLevel = data.alert_level;
 
+  const minimumTimeElapsed = (
+    data.worldtime > data.distresstimelock);
+
   const canAnnounce = (
     data.endtime < data.worldtime);
 
+  const roundends = data.roundends;
+
   const canEvac = (
     evacstatus === 0,
     AlertLevel >= 2);
 
+  const canDistress = (
+    AlertLevel === 2 && !data.distresscd && minimumTimeElapsed);
```

Comment:
`!data.distresscd`
this will not work you need to do it like with the tablet (it's not a value that goes down, it's a timestamp that does not change until the cooldown is restarted)